### PR TITLE
KV DynamoDB metrics

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: pkg/kv/dynamodb/store.go [hunk 1]
```diff
@@ -195,12 +195,15 @@ func (s *Store) Get(ctx context.Context, partitionKey, key []byte) (*kv.ValueWit
 		return nil, kv.ErrMissingKey
 	}
 
+	start := time.Now()
 	result, err := s.svc.GetItemWithContext(ctx, &dynamodb.GetItemInput{
 		TableName:      aws.String(s.params.TableName),
 		Key:            s.bytesKeyToDynamoKey(partitionKey, key),
 		ConsistentRead: aws.Bool(true),
 	})
+	dynamoRequestDuration.WithLabelValues("GetItem").Observe(time.Since(start).Seconds())
```
Comment (itaiad200) [inline pkg/kv/dynamodb/store.go:204]: Why is the consumed capacity not counted for in this flow?

**Review by itaiad200 — APPROVED** [review]
LGTM
Just a single question

File: pkg/kv/dynamodb/store.go (hunk 1 above)
Comment (nopcoder) [reply pkg/kv/dynamodb/store.go]: Missed it - thanks!

**Review by itaiad200 — APPROVED** [review]

File: pkg/kv/dynamodb/store.go [hunk 2]
```diff
@@ -351,6 +382,9 @@ func (s *Store) DropTable() error {
 	_, err := s.svc.DeleteTable(&dynamodb.DeleteTableInput{
 		TableName: &s.params.TableName,
 	})
+	if err != nil {
+		dynamoFailures.WithLabelValues("DeleteTable").Inc()
```
Comment (N-o-Z) [inline pkg/kv/dynamodb/store.go:386]: This is an internal function for testing. Not sure we need to collect metrics on it.

**Review by N-o-Z — COMMENTED** [review]
@nopcoder Thanks!
Do we need to add documentation for it?

File: pkg/kv/dynamodb/store.go (hunk 2 above)
Comment (nopcoder) [reply pkg/kv/dynamodb/store.go:386]: true, will remove it

**Comment by nopcoder** [comment]
> @nopcoder Thanks!
> Do we need to add documentation for it?

good point, ill add pr for the documentation

**Pull request merged**